### PR TITLE
_JAVA_OPTIONS

### PR DESCRIPTION
**Pull request by Ladvace**

### CODE REVIEW

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5473630&bid=17938933)

File: src/common/modals/JavaOptionsFixModal.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+          // Removes the _JAVA_OPTIONS and sets it to empty in the user registry to apply it without rebooting
+          await promisify(childProcess.exec)(
+            'SETX _JAVA_OPTIONS "" & REG delete HKCU\\Environment /F /V _JAVA_OPTIONS'
+          );
+        } catch (e) {
+          console.err(e);
+        }
+      }
+    } else {
+      try {
```
Comment (killpowa) [inline src/common/modals/JavaOptionsFixModal.js:50]: Codacy Issue found: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5473630)

**Comment by blarfoon** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5473630&bid=17938933)

File: src/common/modals/JavaOptionsFixModal.js (hunk 1 above)
Comment (blarfoon) [inline src/common/modals/JavaOptionsFixModal.js:50]: Codacy Issue found: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5473630)

**Pull request closed without merging**